--- mailutils-3.1.1/lib/Makefile.am.orig	2016-12-03 16:51:47.000000000 +0100
+++ mailutils-3.1.1/lib/Makefile.am	2017-02-09 20:41:05.002880589 +0100
@@ -35,7 +35,7 @@
  strexit.c\
  mu_umaxtostr.c\
  mu_umaxtostr.h
-libmuaux_la_LIBADD=gnu/libgnu.la
+libmuaux_la_LIBADD=gnu/libgnu.la ../libmailutils/libmailutils.la
 libmuaux_la_LDFLAGS = -version-info @VI_CURRENT@:@VI_REVISION@:@VI_AGE@
 
 libmutcpwrap_a_SOURCES = tcpwrap.c
--- mailutils-3.1.1/configure.ac.orig	2017-02-11 21:54:08.860857465 +0100
+++ mailutils-3.1.1/configure.ac	2017-02-12 09:59:42.887027050 +0100
@@ -190,8 +190,6 @@
 case $status_tcpwrap in
  yes) AC_CHECK_LIB(wrap, main,,
                    [AC_MSG_ERROR([Required library libwrap not found])])
-      AC_CHECK_LIB(nsl, main,
-                   [TCPWRAP_LIBRARIES=-lnsl])
       AC_CHECK_HEADERS(tcpd.h,,
                        [AC_MSG_ERROR([Required header tcpd.h not found])])
       status_tcpwrap=yes
@@ -201,7 +199,6 @@
       AC_CHECK_LIB(wrap, main,
                    [status_tcpwrap=yes],
                    [status_tcpwrap=no])
-      AC_CHECK_LIB(nsl, main, [TCPWRAP_LIBRARIES=-lnsl])
       AC_CHECK_HEADERS(tcpd.h,
                        [status_tcpwrap=yes],
                        [status_tcpwrap=no])
--- mailutils-3.4/libmu_cpp/Makefile.am.orig	2017-06-08 15:08:20.000000000 +0200
+++ mailutils-3.4/libmu_cpp/Makefile.am	2017-12-10 21:03:12.276619950 +0100
@@ -42,6 +42,6 @@
  stream.cc\
  url.cc
 
-libmu_cpp_la_LIBADD = @MU_COMMON_LIBRARIES@
+libmu_cpp_la_LIBADD = $(top_builddir)/libmailutils/libmailutils.la $(top_builddir)/libmu_sieve/libmu_sieve.la $(top_builddir)/libproto/pop/libmu_pop.la @MU_COMMON_LIBRARIES@
 libmu_cpp_la_LDFLAGS = -rpath $(libdir)  -version-info @VI_CURRENT@:@VI_REVISION@:@VI_AGE@
 
--- mailutils-3.4/Makefile.am.orig	2017-10-18 10:51:32.000000000 +0200
+++ mailutils-3.4/Makefile.am	2017-12-10 21:26:46.673270471 +0100
@@ -96,14 +96,12 @@
  libproto\
  testsuite\
  lib\
- $(LIBMU_CPP_DIR)\
  $(GINT_DIR)\
  $(LIBMU_SCM_DIR)\
  $(LIBMU_DBM_DIR)\
  libmu_sieve\
  $(PYTHON_DIR)\
  doc\
- examples\
  mu\
  $(SCHEME_DIR)\
  $(FRM_DIR)\
@@ -113,6 +111,8 @@
  $(MAIL_DIR)\
  $(SIEVE_DIR)\
  $(MESSAGES_DIR)\
+ $(LIBMU_CPP_DIR)\
+ examples\
  $(COMSAT_DIR)\
  $(READMSG_DIR)\
  $(DOTLOCK_DIR)\
