--- mailutils-3.8/libmailutils/wordsplit/wordsplit.h.orig	2019-07-23 06:36:52.000000000 +0200
+++ mailutils-3.8/libmailutils/wordsplit/wordsplit.h	2019-11-08 21:28:14.865027949 +0100
@@ -19,6 +19,10 @@
 
 #include <stddef.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 typedef struct wordsplit wordsplit_t;
 
 /* Structure used to direct the splitting.  Members marked with [Input]
@@ -303,4 +307,8 @@
 
 void wordsplit_clearerr (wordsplit_t *ws);
 
+#ifdef __cplusplus
+}
+#endif
+
 #endif
