--- mailutils-3.18/libmailutils/diag/debug.c.orig	2025-01-02 14:36:30.000000000 +0100
+++ mailutils-3.18/libmailutils/diag/debug.c	2025-01-18 18:48:43.651344459 +0100
@@ -511,7 +511,7 @@ mu_debug_format_spec (mu_stream_t str, c
 	{
 	  if (delim)
 	    {
-	      rc = mu_stream_printf (str, delim);
+	      rc = mu_stream_printf (str, "%s", delim);
 	      if (rc)
 		break;
 	    }
@@ -564,7 +564,7 @@ mu_debug_format_spec (mu_stream_t str, c
 	{
 	  if (delim)
 	    {
-	      rc = mu_stream_printf (str, delim);
+	      rc = mu_stream_printf (str, "%s", delim);
 	      if (rc)
 		break;
 	    }
