--- mailutils-3.18/libmailutils/wordsplit/wordsplit.h.orig	2024-10-30 13:28:24.000000000 +0100
+++ mailutils-3.18/libmailutils/wordsplit/wordsplit.h	2025-01-17 06:25:06.521202356 +0100
@@ -19,6 +19,10 @@
 
 #include <stddef.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 typedef struct wordsplit wordsplit_t;
 
 /* Structure used to direct the splitting.  Members marked with [Input]
@@ -319,4 +323,8 @@ enum
 
 extern char const *wordsplit_escape[];
 
+#ifdef __cplusplus
+}
+#endif
+
 #endif
