--- mailutils-3.1.1/configure.ac.orig	2017-02-12 09:59:42.887027050 +0100
+++ mailutils-3.1.1/configure.ac	2017-02-12 11:02:17.426984182 +0100
@@ -506,10 +506,10 @@
   BDB2)   check_bdb 2 db db_open
           test -n "$status_bdb" && status_dbm="$status_dbm,Berkeley DB v. $status_bdb";;
 
-  BDB)    for version in 4 3 2
+  BDB)    for version in 6 5 4 3 2
 	  do
 		case $version in
-		4|3) func=db_create;;
+		6|5|4|3) func=db_create;;
 		2)   func=db_open;;
 		esac
 		check_bdb $version db $func
@@ -523,7 +523,7 @@
 	  name=`expr $1 : 'BDB=\(.*\)'`
   	  case $name in
 	  [[0-9]]*)  check_slackware_bdb $name;;
-	  *)	     for version in 4 3 2
+	  *)	     for version in 6 5 4 3 2
 	             do
 			case $version in
 			4|3) func=db_create;;
